test/src/test/java/org/corfudb/integration/CorfuBrowserIT.java

```diff
@@ -0,0 +1,174 @@
+package org.corfudb.integration;
+
+import com.google.protobuf.UnknownFieldSet;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.corfudb.browser.CorfuBrowser;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.CorfuDynamicRecord;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.runtime.collections.TxBuilder;
+import org.corfudb.test.SampleAppliance;
+import org.corfudb.test.SampleSchema;
+
+
+public class CorfuBrowserIT extends AbstractIT {
+
+    private static String corfuSingleNodeHost;
+
+    private static int corfuStringNodePort;
+
+    private static String singleNodeEndpoint;
+
+    /* A helper method that takes host and port specification, start a single server and
+     *  returns a process. */
+    private Process runSinglePersistentServer(String host, int port) throws
+        IOException {
+        return new AbstractIT.CorfuServerRunner()
+            .setHost(host)
+            .setPort(port)
+            .setLogPath(getCorfuServerLogPath(host, port))
+            .setSingle(true)
+            .runServer();
+    }
+
+    /**
+     * Load properties for a single node corfu server before each test
+     */
+    @Before
+    public void loadProperties() {
+        corfuSingleNodeHost = PROPERTIES.getProperty("corfuSingleNodeHost");
+        corfuStringNodePort = Integer.valueOf(PROPERTIES.getProperty(
+            "corfuSingleNodePort"));
+        singleNodeEndpoint = String.format(
+            "%s:%d",
+            corfuSingleNodeHost,
+            corfuStringNodePort
+        );
+    }
+
+    @Test
+    public void browserTest() throws
```

Comment:
Good tests.
Please add high-level comments describing the test.

Reply:
done